PWGDQ/Core/MCProng.h

```diff
@@ -79,14 +79,15 @@ class MCProng
   MCProng();
   MCProng(int n);
   MCProng(int n, std::vector<int> pdgs, std::vector<bool> checkBothCharges, std::vector<bool> excludePDG,
-          std::vector<uint64_t> sourceBits, std::vector<uint64_t> excludeSource, std::vector<bool> useANDonSourceBitMap);
+          std::vector<uint64_t> sourceBits, std::vector<uint64_t> excludeSource, std::vector<bool> useANDonSourceBitMap, bool fCheckGenerationsInTime = false);
   MCProng(const MCProng& c) = default;
   virtual ~MCProng() = default;
 
   void SetPDGcode(int generation, int code, bool checkBothCharges = false, bool exclude = false);
   void SetSources(int generation, uint64_t bits, uint64_t exclude = 0, bool useANDonSourceBits = true);
   void SetSourceBit(int generation, int sourceBit, bool exclude = false);
   void SetUseANDonSourceBits(int generation, bool option = true);
+  void SetSignalInTime(int generation, bool intime = false); // set variable to check generations in time or back in time (default)
```

Comment:
This function does not need the first parameter (generation), since the rule to go back in time or in time, holds for the entire prong. So please remove it, only the "intime" parameter is used